www/js/types/diaryTypes.ts

```diff
@@ -61,24 +56,27 @@ type CompTripLocations = {
 //  Used for return type of readUnprocessedTrips
 export type UnprocessedTrip = {
   _id: ObjectId;
-  additions: UserInputEntry[];
+  additions: []; // unprocessed trips won't have any matched processed inputs, so this is always empty
   confidence_threshold: number;
   distance: number;
   duration: number;
   end_fmt_time: string;
   end_loc: Point;
   end_local_dt: LocalDt;
-  expectation: any; // TODO "{to_label: boolean}"
-  inferred_labels: any[]; // TODO
-  key: string;
-  locations?: CompTripLocations[];
-  origin_key: string; // e.x., UNPROCESSED_trip
-  source: string;
+  end_ts: number;
+  expectation: { to_label: true }; // unprocessed trips are always expected to be labeled
+  inferred_labels: []; // unprocessed trips won't have inferred labels
+  key: 'UNPROCESSED_trip';
+  locations?: CompositeTripLocation[];
+  origin_key: 'UNPROCESSED_trip';
+  sections: SectionData[];
+  source: 'unprocessed';
+  start_fmt_time: string;
```

Comment:
That is just how unprocessed trips have always been represented.

This isn't a "real" key because it doesn't come from the server. It's a client-side construction of something similar to a composite_trip object, but it wouldn't really be accurate to call it a de facto composite_trip. So it doesn't get the `analysis/composite_trip` key.

Somewhere else in the codebase, we identify draft trips by seeing if the key includes 'UNPROCESSED'.